### PR TITLE
Awesome books es6

### PR DESCRIPTION
**Pull request by Osoro254Alex**
I did the following on Awesome Book Repo:

- I cloned the Awesome Book Repo.
- Created new branch Awesome-books-Es6.
- Add README.md file.
- Add index.css file.
- Add index.html file
- Add modules folder.
- Add Module files of JavaScript in it.
- Add images folder.
- Add .gitignore file.
- Add .eslintignore file.
- Add Linter files and Fix Linter Errors.

### CODE REVIEW

File: modules/main.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+books.removeButton();
+
+books.displayBooks();
+
+getDate();
+
+navLinks();
+
+document.addEventListener('DOMContentLoaded', books.displayBooks());
```
Comment (ShadyShawkat) [inline modules/main.js]: - [ ] Kindly fix the following behaviour:
   - On page reload when there're books in local storage, each book is rendered twice (See attached below)
   
image

File: modules/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+/* eslint-disable */
+
```
Comment (ShadyShawkat) [inline modules/index.js]: - [ ] Please refrain from using `eslint-disable` on the whole file, as it defies the purpose of using linters in the first place.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Osoro254Alex, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@ShadyShawkat 
To refrain from using eslint-disable
To fix books from being rendered twice from local storage on reload page.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Osoro254Alex       :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

- [ ] [OPTIONAL] I have realized that you have numerous comments in your code base. This is something I would strongly discourage you from continuing doing mainly because many comments make your code base look cluttered and messy and the extra lines mean extra bandwidth as more space will have been used up when loading the website. *I know this does not really apply here but picture a website which has thousands of comments)

The idea behind keeping your code logic as simple as possible till a point where you do not need comments to note what a specific line or block of code does is perfectly explained by KISS (Keep it simple stupid). For some insights, I highly recommend you to read [this article](https://www.interaction-design.org/literature/topics/keep-it-simple-stupid) I use and hopefully afterward you will remove the many comments that are in your code base.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@Nemwel-Boniface 

- Told me how I can Improve more.
- Approved.

**Pull request merged**